docs/getting-started/prefect-explained.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Prefect Explained
+description: Learn core Prefect concepts.
+tags:
+    - getting started
+    - explanation
+    - overview
+search:
+  boost: 2
+---
+
+# Prefect Explained
+
+Prefect is a Pythonic framework for orchestrating and observing workflows.
+
+You install the [`prefect` package](/getting-started/installation/) to gain access to the Prefect client that contains the Python SDK and CLI commands.
+
+The Prefect client talks to a Prefect server - either the managed [Prefect Cloud platform](/cloud/) or a [self-hosted server instance](/guides/host/) backed by SQLite or PostgreSQL.
+
+Prefect Cloud provides [free and paid plans](https://www.prefect.io/pricing) with additional features useful for teams and production workloads.
+
+Both a self-hosted server and Prefect Cloud provide a UI for visualizing and managing your workflows.
+
+![Cloud Dashboard](/img/ui/cloud-dashboard.png)
+
+Here are the three major concepts you need to know to build a production workflow with Prefect:
+
+- Flows
+- Deployments
+- Work pools
+
+## Flows
+
+A Prefect [flow](/concepts/flows/) is a Python function that defines a workflow and is decorated by `@flow`.
+When this function runs, Prefect can track the state of the workflow and visualize it in the UI.
+
```

Comment:
Maybe put `As shown below, when a flow is referenced as the entrypoint in a deployment it can be scheduled.` after the screenshot? It feels like the screenshot references `Prefect can track the state of the workflow and visualize it in the UI.`